src/tex/sections/patterns_in_wrong_questions.tex

```diff
@@ -0,0 +1,4 @@
+The models seem to not be able to answer questions that require specific knowledge (e.g., safety profile of chemicals) and reasoning (e.g., oxidation number of a specific element, the number of chiral centers in an organic molecule or identifying a symmetry group).
+Models also struggle with identifying the relevant chemical trends (e.g., ordering the entropy of compounds).
+
+\glspl{llm} also struggle with open-ended questions. Albeit in our study, we only investigated numerical open-ended questions, thus a part of these would require multi-step calculations, which can be challenging for \glspl{llm}.\autocite{zhou2023llm_arithmetics}
```

Comment:
```suggestion
The models seem to not be able to answer questions that require specific knowledge (e.g., safety profile of chemicals) and multistep reasoning (e.g., oxidation number of a specific element, the number of chiral centers in an organic molecule or identifying a symmetry group). A common characteristic of many questions on which the models failes is that they require mapping a text representation of a compound to a property of the two- or three-dimensional structure of the compound. 

\glspl{llm} also struggle with open-ended questions. Albeit in our study, we only investigated numerical open-ended questions, thus a part of these would require multi-step calculations, which can be challenging for \glspl{llm}.\autocite{zhou2023llm_arithmetics}
```

the entropy thing I'd remove or rephrase as entropy is relative and the question we had was not clear about this. 

what questions do you refer to with "open-ended questions"?

Reply:
the ones that are not mcq

Reply:
> the ones that are not mcq

but then I'd be specific and just mention them